src/utils/types.js

```diff
@@ -58,3 +58,11 @@ export function isFunction(value) {
 export function isBoolean(value) {
   return typeof value === 'boolean';
 }
+
+/**
+ * @param {?*} data
+ * @return {boolean}
+ */
+export function isMessage(data) {
```

Comment:
`Message` is a kind of general name, could we call this `isProtoMessage` to be more specific?